server/core/src/main/java/dev/slimevr/protocol/rpc/serial/RPCSerialHandler.java

```diff
@@ -38,6 +38,11 @@ public RPCSerialHandler(RPCHandler rpcHandler, ProtocolAPI api) {
 				RpcMessage.SerialTrackerFactoryResetRequest,
 				this::onSerialTrackerFactoryResetRequest
 			);
+		rpcHandler
+			.registerPacketListener(
+				RpcMessage.SerialTrackerGetWifiScanRequest,
+				this::onSerialTrackerGetWifiScanRequest
+			);
```

Comment:
Why is this just randomly in the middle?

Reply:
I did try to keep serial functions in one part of the code.
Not one at end and the other at the start.